src/ReactCardFlip.tsx

```diff
@@ -16,14 +16,28 @@ const ReactCardFlip: React.FC<ReactFlipCardProps> = (props) => {
     flipSpeedFrontToBack,
     flipSpeedBackToFront,
     infinite,
-  } = props;
-
-  const [isFlipped, setFlipped] = useState(props.isFlipped);
+    isFlipped,
```

Comment:
This conflicts with the internal state variable defined on line 35.

Reply:
I think it should be fixed now.